libs/langchain/langchain/schema/output_parser.py

```diff
@@ -329,6 +334,69 @@ async def atransform(
             yield chunk
 
 
+class BaseCumulativeTransformOutputParser(BaseTransformOutputParser[T]):
+    """Base class for an output parser that can handle streaming input."""
+
+    diff: bool = False
+
+    def _diff(self, prev: Optional[T], next: T) -> T:
```

Comment:
Perhaps document this method?

Do you forsee a situation in which `_diff` is defined but the user passes `diff=False` ?